### ISSUE TITLE
Multiple keyframes creating an infinite loop

### ISSUE THREAD

**Issue by firynth** [Keyframes/jQuery.Keyframes#68]
```javascript
function animatePrinny() {
  $("#prinnyDOOD").playKeyframe({
    name: 'slideUpHalf',
    duration: "3000ms",
    iterationCount: 1,
    "complete" : function(){
        setTimeout(function() {
            $("#exclamation").show();
        }, 200);
        loadAudio("tindeck_1.mp3", function() {
            $("#exclamation").hide();
            $("#prinnyDOOD").playKeyframe({
                name: 'slideUp',
                duration: '500ms',
                iterationCount: 1,
                "complete" : function() {
                    loadAudio("thanksdood.wav", function() {
                        $("#prinnyDOOD").playKeyframe({
                            name: 'slideDown',
                            duration: "500ms",
                            "complete" : function() {
                                playAnimation();
                                $("#prinnyDOOD").resetKeyframe();
                            }
                        });
                    });
                }
            });
        });
      }
  });
}
```

This code causes the animations and the sounds to play infinitely, causing the audio files to get louder and louder and louder. It didn't happen before the latest update.

**Comment by krazyjakee** [comment]
Thanks for the report. Could you please create a minimal example is [jsfiddle](https://jsfiddle.net/) or [codepen](https://codepen.io/) so I can recreate this issue?

**Comment by krazyjakee** [comment]
The fix is in 1.0.3 now deployed.
Check this working example: https://jsfiddle.net/x3ou8ejv/

**Comment by firynth** [comment]
Looks like that fixed the issue. Thanks!
